app/controllers/practices_controller.rb

```diff
@@ -1,25 +1,20 @@
 class PracticesController < ApplicationController
   include CropperUtils
-  before_action :set_practice, only: [:show, :edit, :update, :destroy, :planning_checklist,
+  before_action :set_practice, only: [:show, :edit, :update, :destroy,
                                       :commit, :committed, :highlight, :un_highlight, :feature,
                                       :un_feature, :favorite, :instructions, :overview, :origin,
                                       :collaborators, :impact, :resources, :documentation,
                                       :departments, :timeline, :risk_and_mitigation, :contact,
-                                      :checklist, :publication_validation, :adoptions,
+                                      :publication_validation, :adoptions,
                                       :create_or_update_diffusion_history, :implementation, :introduction, :about]
-  before_action :set_facility_data, only: [:show, :planning_checklist]
```

Comment:
we still want the next 4 `before_action`s for `:set_facility_data`, `:set_office_data`, `:set_visn_data`, :set_initiating_facility_other` for `:show`

Reply:
Got it - thx again!